x-pack/solutions/security/plugins/security_solution/public/management/components/console/components/console_state/state_update_handlers/handle_input_area_state.ts

```diff
@@ -77,6 +78,18 @@ export const handleInputAreaState: ConsoleStoreReducer<InputAreaStateAction> = (
               id: uuidV4(),
               input: payload.command,
               display: payload.display ?? payload.command,
+              // We only store the `value` and `valueText`. `store` property of each argument's state
+              // is component instance specific data.
+              argState: Object.entries(payload.argState || {}).reduce(
```

Comment:
Just FYI (for the future, case its needed). The console history also supports migration if we ever need it. I have used it once in the past (version is currently at `2`), but for this change here I don't think we need it since the data is optional.

https://github.com/elastic/kibana/blob/da25d13a2ac5be97216e202bf5d6bda6ff920b3b/x-pack/solutions/security/plugins/security_solution/public/management/components/console/components/console_state/hooks/use_stored_input_history.ts#L79-L93